src/index.html

```diff
@@ -1,14 +1,204 @@
+
+    <p class="paragraph">Everyone is entitled to all the rights and freedoms set forth in this Declaration,
+    without distinction of any kind, such as race, colour, sex, language, religion,
+    political or other opinion, national or social origin, property, birth or other status.
+    Furthermore, no distinction shall be made on the basis of the political,
+    jurisdictional or international status of the country or territory to which a person
+    belongs, whether it be independent, trust, non-self-governing or under any other
+    limitation of sovereignty.
+    </p>
+
+    <h2 class="subtitle">Article 3</h2>
+
+    <p class="paragraph">Everyone has the right to life, liberty and security of person.
+    </p>
+
+    <h2 class="subtitle">Article 4</h2>
+
+    <p class="paragraph">No one shall be held in slavery or servitude; slavery and the slave trade shall be
+    prohibited in all their forms.
+    </p>
+
+    <h2 class="subtitle">Article 5</h2>
+
+    <p class="paragraph">No one shall be subjected to torture or to cruel, inhuman or degrading treatment
+    or punishment.
+    </p>
+
+    <h2 class="subtitle">Article 6</h2>
+
+    <p class="paragraph">Everyone has the right to recognition everywhere as a person before the law.
+    </p>
+
+    <h2 class="subtitle">Article 7</h2>
+
+    <p class="paragraph">All are equal before the law and are entitled without any discrimination to equal
+    protection of the law. All are entitled to equal protection against any
+    discrimination in violation of this Declaration and against any incitement to such
+    discrimination.
+    </p>
+
+    <h2 class="subtitle">Article 8</h2>
+
+    <p class="paragraph">Everyone has the right to an effective remedy by the competent national tribunals
+    for acts violating the fundamental rights granted him by the constitution or by law.
+    </p>
+
+    <h2 class="subtitle">Article 9</h2>
+
+    <p class="paragraph">No one shall be subjected to arbitrary arrest, detention or exile.
+    </p>
+    <h2 class="subtitle">Article 10</h2>
+
+    <p class="paragraph">Everyone is entitled in full equality to a fair and public hearing by an independent
+    and impartial tribunal, in the determination of his rights and obligations and of any
+    criminal charge against him.
+    </p>
+
+    <h2 class="subtitle">Article 11</h2>
+
+    <p class="paragraph">
+      <ol>
+        <li>Everyone charged with a penal offence has the right to be presumed
+    innocent until proved guilty according to law in a public trial at which he
+    has had all the guarantees necessary for his defence.
+        </li>
+
+        <li>No one shall be held guilty of any penal offence on account of any act or
+    omission which did not constitute a penal offence, under national or
+    international law, at the time when it was committed. Nor shall a heavier
+    penalty be imposed than the one that was applicable at the time the penal
+    offence was committed.
+        </li>
```

Comment:
put the text on the newline and use soft tabs (2 spaces) for indentation everywhere

```suggestion
        <li>
           No one shall be held guilty of any penal offence on account of any act or
           omission which did not constitute a penal offence, under national or
           international law, at the time when it was committed. Nor shall a heavier
           penalty be imposed than the one that was applicable at the time the penal
           offence was committed.
      </li>
```